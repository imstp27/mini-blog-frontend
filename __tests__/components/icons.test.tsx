import React from 'react'
import { CircleOutlined } from '@components/icons'
import { shallow, mount, ShallowWrapper } from 'enzyme'

describe("Card component", () => {
    let wrapper: ShallowWrapper
    beforeAll(() => {
        wrapper = shallow(<CircleOutlined />);
    })
    test("it's renders", () => {
        expect(wrapper.exists()).toBe(true);
    });
    test("it's must match snapshot", () => {
        expect(wrapper).toMatchSnapshot();
    });
});