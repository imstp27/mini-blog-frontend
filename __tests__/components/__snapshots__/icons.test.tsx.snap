// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component it's must match snapshot 1`] = `
<svg
  height="1em"
  viewBox="0 0 34.76 34.76"
  width="1em"
  x="0px"
  y="0px"
>
  <g>
    <g>
      <path
        d="M17.38,34.76c-9.583,0-17.379-7.796-17.379-17.38S7.797,0,17.38,0s17.379,7.796,17.379,17.38S26.963,34.76,17.38,34.76z M17.38,6.495c-6.002,0-10.885,4.882-10.885,10.885s4.883,10.885,10.885,10.885s10.885-4.882,10.885-10.885 S23.382,6.495,17.38,6.495z"
      />
    </g>
  </g>
</svg>
`;
